frb_spectral_search/spectral_search.py

```diff
@@ -910,28 +909,28 @@ def acf_lombscarg(spectra, noise_spec, plot_range = None):
             ax1.xaxis.set_minor_locator(MultipleLocator(1))
     ax1.set_xlabel( "frequency lag (24 kHz channels)", fontsize =16 )
     plt.show()
-    
+
 def gauss_subtract_wo_and_replace_zero(spec , wid):
     #returns spectrum minus the smoothed gaussian with zeros uneffected
     bigsig = gaussian(np.linspace(-100,100,300), 0, wid)
```

Comment:
I don't understand this, is your channel spacing (100 - -100) / 300 = 0.67 channels for the Gaussian smoothing? Shouldn't the spacing be exactly 1 channel so that `wid` can be interpreted as the number of channels you smooth together?